extensions/wikia/ExactTargetUpdates/maintenance/ExactTargetUpdateUserEditsPerWikiMaintenance.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Maintenance script for updating number of edits made by user on a wiki
+ * in ExactTarget database
+ *
+ * @package MediaWiki
+ * @addtopackage maintenance
+ *
+ * @author Kamil Koterba <kamil@wikia-inc.com>
+ *
+ */
+
+require_once( __DIR__ . "/../../../../maintenance/commandLine.inc" );
+
+class ExactTargetUpdateUserEditsPerWikiMaintenance {
+
+	private $aBotsList = NULL;
+
+	/**
+	 * Maintenance script entry point.
+	 * Gathering user edits data from last day and adding update task to job queue.
+	 * Script skips old TempUser relicts and bot accounts.
+	 */
+	public function execute() {
+		global $wgDWStatsDB;
+		// Get DB
+		$oStatsDBr = wfGetDB( DB_SLAVE, array(), $wgDWStatsDB );
```

Comment:
The code currently switches between using the old `array()` and the newer square brackets. It's better to be consistent throughout the extension at least one way or the other.